mreg/api/permissions.py

```diff
@@ -341,11 +341,17 @@ def has_ipaddress_permission(self, request: Request, view: ListCreateAPIView, va
         if (user.is_mreg_superuser_or_admin or user.is_mreg_network_admin):
             return True
 
-        if isinstance(view, (
-                        mreg.api.v1.views.HostList,
-                        mreg.api.v1.views.HostDetail,
-                        mreg.api.v1.views.IpaddressList,
-                        mreg.api.v1.views.PtrOverrideList)):
+        if not isinstance(
+            view, 
+            (
+                mreg.api.v1.views.HostList,
+                mreg.api.v1.views.HostDetail,
+                mreg.api.v1.views.IpaddressDetail,
+                mreg.api.v1.views.IpaddressList,
+                mreg.api.v1.views.PtrOverrideDetail,
+                mreg.api.v1.views.PtrOverrideList
+            )
+        ):
```

Comment:
Can we remove this check? As long as we have an `"ipaddress"` that we mutate or assign somehow, we might want to check if the user is allowed to use such an IP? 

Then again, I'm sure doing that opens the door for some strange bugs down the road because we make too broad of an assumption about the semantics of such an `"ipaddress"` field.